WalletWasabi/Blockchain/Keys/KeyManager.cs

```diff
@@ -701,10 +701,6 @@ public void SetIcon(WalletType type)
 	public void SetAnonScoreTargets(int minAnonScoreTarget, int maxAnonScoreTarget)
 	{
 		MinAnonScoreTarget = minAnonScoreTarget;
-		if (MaxAnonScoreTarget < MinAnonScoreTarget)
-		{
-			throw new ArgumentException($"{nameof(maxAnonScoreTarget)} should be greater.", nameof(maxAnonScoreTarget));
-		}
 		MaxAnonScoreTarget = maxAnonScoreTarget;
 		ToFile();
 	}
```

Comment:
The problem is that it checks the properties instead of the parameters.

This would work perfectly:
```
public void SetAnonScoreTargets(int minAnonScoreTarget, int maxAnonScoreTarget)
{
	if (maxAnonScoreTarget < minAnonScoreTarget)
	{
		throw new ArgumentException($"{nameof(maxAnonScoreTarget)} should be greater.", nameof(maxAnonScoreTarget));
	}

    MinAnonScoreTarget = minAnonScoreTarget;
	MaxAnonScoreTarget = maxAnonScoreTarget;

	ToFile();
}
```

Reply:
I don't know how those sliders work but I think that check already exists in UI and thus no need for that when setting those anon score targets.

Reply:
Yes, you are right. Ui does not allow setting it incorrectly, right now. 

But from a coding perspective, that exception makes sense. If someone uses this method through the UI, they cannot make any mistakes because the UI makes sure the values are correct. But if someone uses it from somewhere in the code in the future, and if there is no check, it can cause errors/bugs.

Reply:
Roland is right, the code should check the parameters.

Reply:
Done.